plugins/keyboard-navigation/test/navigation_modify_test.mocha.js

```diff
@@ -10,25 +10,7 @@ const {Navigation} = require('../src/navigation');
 const assert = chai.assert;
 
 suite('Insert/Modify', function() {
-  /**
-   * Captures the strings sent to console.warn() when calling a function.
-   * Copies from core.
-   * @param {function} innerFunc The function where warnings may called.
-   * @return {string[]} The warning messages (only the first arguments).
-   */
-  function captureWarnings(innerFunc) {
-    const msgs = [];
-    const nativeConsoleWarn = console.warn;
-    try {
-      console.warn = function(msg) {
-        msgs.push(msg);
-      };
-      innerFunc();
-    } finally {
-      console.warn = nativeConsoleWarn;
-    }
-    return msgs;
-  }
+
```

Comment:
We need to wait till after the dev-tools plugin is published on npm with the new changes, before removing the helper here. We publish every week. 

After it is published, removing the helper here would be the next step, along with importing the captureWarnings into this file with: 
```
const {testHelpers} = require('@blockly/dev-tools');
// ...
const {captureWarnings} = testHelpers
```